test/e2e/common/node/downwardapi.go

```diff
@@ -293,7 +294,9 @@ var _ = SIGDescribe("Downward API [Serial] [Disruptive] [NodeFeature:DownwardAPI
 
 	ginkgo.Context("Downward API tests for hugepages", func() {
 		ginkgo.BeforeEach(func() {
-			e2eskipper.SkipUnlessFeatureGateEnabled(kubefeatures.DownwardAPIHugePages)
+			if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DownwardAPIHugePages) {
```

Comment:
Currently, it will fail for the nil featureGates. Is there a way to check the current featureGates status of apiserver and kubelet? I have not found the method and so I use the default.(It will skip if it is alpha or the default value is changed.)

Or we should just remove it.

Reply:
> Is there a way to check the current featureGates status of apiserver and kubelet?

No. If that is needed, then the test description needs a "Feature:XYZ" tag and tests must be skipped or enabled depending on the cluster configuration.

> Or we should just remove it.

That is the right choice if there never is a situation where the test must be skipped. I don't know whether that is the case.

Reply:
Updated